docs/3-tutorials/04-genesis-creation.md

```diff
@@ -0,0 +1,129 @@
+# Genesis creation
+
+Horizen  initial state is not empty, but reflects the data present in both previous Horizen's chains: ZEND Mainchain and EON.
+
+The pre-calculated initial state, mandatory to start a new collator node,  is available for download at this URL: [TODO]
+
+To verify its correctness, anyone can follow the instructions in this tutorial and regenerate the same data.
+
+## Prerequisites:
+
+The following software is needed:
+
+1. Fully synched ZEND Mainchain node with dump support enabled:
+
+    - Donwload latest ZEND release from here: https://github.com/HorizenOfficial/zen/releases
+    - Follow the instructions here https://github.com/HorizenLabs/migration-tools/tree/main/snapshot to patch the ZEND node in order to have dump support enabled
```

Comment:
not sure if this is expected, but this link shows 404 page for me.

Reply:
it's because is still private.
we have to open it before announcing the migration - cc @lander86